docs/02.API-REFERENCE.md

```diff
@@ -9560,8 +9838,8 @@ jerry_realm_set_this (jerry_value_t realm_value, jerry_value_t this_value)
 - `realm_value` - realm value
 - `this_value` - new this value
 - return
-  - type error - if realm_value is not a realm or this_value is not object
-  - true - otherwise
+  - type error, if realm_value is not a realm or this_value is not object
+  - true, otherwise
```

Comment:
AFAIK, this change ins not related to the PR.

Reply:
I would not create another pr for this small change.

Reply:
AFIK: All of the return descriptions should be updated not just this one.

Reply:
Ok, I restored it.